developing-with-classicpress/index.md

```diff
@@ -0,0 +1,6 @@
+# Developing With ClassicPress
+
+- <span style="font-size: 150%">[Codex](https://codex.classicpress.net/)</span><!--this should link to the new, not-yet-existing DOC for Code page-->
+- <span style="font-size: 150%">[Security Page](https://docs.classicpress.net/developing-with-classicpress/security-page/)</span><!--this will only work after renaming developing-classicpress to developing-with-classicpress-->
```

Comment:
```suggestion
- <span style="font-size: 150%">[Security Page](https://docs.classicpress.net/developing-with-classicpress/security-page/)</span>
```

Reply:
Agreed. Left the comment only for info purpose.